lightningd/notification.c

```diff
@@ -327,6 +327,38 @@ void notify_channel_state_changed(struct lightningd *ld,
 	plugins_notify(ld->plugins, take(n));
 }
 
+static void we_lost_state_notification_serialize(struct json_stream *stream,
+						const struct channel_id *cid,
+						const struct short_channel_id *scid)
+{
```

Comment:
I prefer a `"lost_state": true` in listpeerchannels, and implementing the wait api for listpeerchannels?  That's more reliable in general, though implementing the wait api for listpeerchannels is a big job, so just poll for now.